src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -84,12 +84,26 @@ private static boolean isValidPath(Path filePath) {
         return filePath.toString().endsWith(".txt");
     }
     
+    /**
+     * Checks if the storage file was deleted while program is running. 
+     * @throws FileDeletedException if storage file was deleted.
+     */
     public void checkFileValidity() throws FileDeletedException{
     	if (!(new File(path.toString()).exists())){
     		throw new FileDeletedException("Storage file has been deleted. Creating new storage file."); 
     	}
```

Comment:
okay, I am a bit lost here. So, what happens in the else or case.. that is normal route.. there is no normal return statement in this method?